Dashboard docs refactor (#674)
dotnet/docs-aspire: docs/fundamentals/dashboard/overview.md

---
title: .NET Aspire dashboard overview
description: Overview of .NET Aspire dashboard and getting started.
ms.date: 03/13/2024
ms.topic: reference
---

> # .NET Aspire dashboard overview

Reviewer: This could do with a dashboard architecture section with this diagram:

![image](https://github.com/dotnet/docs-aspire/assets/95136/a1316bd4-fb78-4815-a915-919d5c18d872)

Author: I don't think this is appropriate for an overview page.

> [.NET Aspire](../../get-started/aspire-overview.md) project templates offer a sophisticated dashboard for comprehensive app monitoring and inspection. This dashboard allows you to track closely various aspects of your application, including logs, traces, and environment configurations, in real-time. It's purpose-built to enhance the local development experience, providing an insightful overview of your app's state and structure.

Reviewer 2: ```suggestion
[.NET Aspire](../../get-started/aspire-overview.md) project templates offer a sophisticated dashboard for comprehensive app monitoring and inspection. This dashboard allows you to closely track various aspects of your app, including logs, traces, and environment configurations, in real-time. It's purpose-built to enhance the local development experience, providing an insightful overview of your app's state and structure.
```

## Using the dashboard with Aspire projects

> The dashboard is integrated into the Aspire **AppHost**. During development the dashboard is autommatically launched when you start the project. It's configured to display the Aspire app's resources and telemetry.

Reviewer 2: ```suggestion
The dashboard is integrated into the .NET Aspire **AppHost**. During development the dashboard is automatically launched when you start the project. It's configured to display the .NET Aspire app's resources and telemetry.
```

:::image type="content" source="media/explore/projects.png" lightbox="media/explore/projects.png" alt-text="A screenshot of the .NET Aspire dashboard Resources page.":::

> [Explore dashboard features](explore.md) to find out more about using dashboard during Aspire development.

Reviewer 2: ```suggestion
For more information about using the dashboard during .NET Aspire development, see [Explore dashboard features](explore.md).
```

## Standalone mode

The .NET Aspire dashboard is also shipped as a docker image and can be used standalone, without the rest of .NET Aspire. The standalone dashboard provides a great UI for viewing telemetry and can be used by any application.

```bash
docker run --rm -it -p 18888:18888 -p 4317:18889 -d --name aspire-dashboard \

>     mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.6

Author: The content (especially in configuration doc) is for preview 6. I'll update the PR to say this should be merged after P6 ships.

```

The preceeding docker command:

> - Starts a container from the `mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.6` image.

Reviewer: ```suggestion
- Starts a container from the `mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.5` image.
```

Reviewer: I see all of your security docs are based on p6
